### DISCUSSION TITLE
Translate SmartFolder?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#18289] [Q&A]
There are some new strings on Weblate for something called SmartFolder. The way it is written makes me wonder if "SmartFolder" should be treated as a brand name and left untranslated? In other strings, e.g. "Add smart folder", translating is more straightforward.

**Comment by DmitryAlexei** [comment]
It is not a brand name. From my point of view, the translation should be the same everywhere for better understanding.
